client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -95,38 +152,50 @@ export class CommentDetailAuthor extends Component {
 					<div className="comment-detail__author-more-element">
 						<Gridicon icon="mail" />
 						<span>
-							{ authorEmail }
+							{ authorEmail || <em>{ translate( 'No email address' ) }</em> }
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Your email address.', 'NULL', { context: 'NULL'} )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/comments/user-info)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).